web-common/src/features/dashboards/time-controls/comparison-pill/ComparisonPill.svelte

```diff
@@ -54,12 +54,6 @@
     start: Date,
     end: Date,
   ) {
-    if (!showTimeComparison) {
```

Comment:
Moved to `setSelectedComparisonRange` to make sure this action leads to a single store update.